L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCTriggerCells.cc

```diff
@@ -74,6 +81,7 @@ DEFINE_EDM_PLUGIN(HGCalTriggerNtupleFactory,
 HGCalTriggerNtupleHGCTriggerCells::
 HGCalTriggerNtupleHGCTriggerCells(const edm::ParameterSet& conf):HGCalTriggerNtupleBase(conf),
   fill_simenergy_(conf.getParameter<bool>("FillSimEnergy")),
+  fill_truthmap_(conf.getParameter<bool>("FillTruthMap")),
```

Comment:
This parameter should be added to the ntuple cfi file and initialized to `False`

Reply:
Sorry for the oversight. Fixed.